app/src/main/java/eu/kanade/tachiyomi/smartsearch/SmartSearchEngine.kt

```diff
@@ -54,18 +56,24 @@ class SmartSearchEngine(
     }*/
 
     suspend fun normalSearch(source: CatalogueSource, title: String): SManga? {
+        val titleNormalized = title.toNormalized()
         val eligibleManga = supervisorScope {
             val searchQuery = if (extraSearchParams != null) {
-                "$title ${extraSearchParams.trim()}"
-            } else title
-            val searchResults = source.fetchSearchManga(1, searchQuery, source.getFilterList()).toSingle().await(Schedulers.io())
+                "$titleNormalized ${extraSearchParams.trim()}"
+            } else titleNormalized
+            val searchResults =
+                source.fetchSearchManga(1, searchQuery, source.getFilterList()).toSingle()
+                    .await(Schedulers.io())
 
             if (searchResults.mangas.size == 1) {
-                return@supervisorScope listOf(SearchEntry(searchResults.mangas.first(), 0.0))
+                val onlyManga = searchResults.mangas.first()
+                onlyManga.setTitleNormalized()
+                return@supervisorScope listOf(SearchEntry(onlyManga, 0.0))
             }
 
             searchResults.mangas.map {
-                val normalizedDistance = normalizedLevenshtein.similarity(title, it.title)
+                it.setTitleNormalized()
+                val normalizedDistance = normalizedLevenshtein.similarity(titleNormalized, it.title)
```

Comment:
Still, I wouldn't change the title in migration, just the search term.
```suggestion
                val normalizedDistance = normalizedLevenshtein.similarity(titleNormalized, it.title.toNormalized())
```

Reply:
Done
I just added the normalization when clicking on manga title and in manual migration, tell me if you don't like it ([b100e69](https://github.com/Jays2Kings/tachiyomiJ2K/pull/1175/commits/b100e6981ced4412aeb0a78e441acf49a2852b92))